app/models/replication.rb

```diff
@@ -0,0 +1,4 @@
+class Replication < ActiveRecord::Base #ResourceUsage why doesn't deriving work?
```

Comment:
oh, i think you don't actually need a resourceusage model. replication can be a subset of debit transaction.